pkg/server/plugin/upstreamauthority/gcpcas/gcpcas_test.go

```diff
@@ -0,0 +1,58 @@
+package gcpcas
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/spiffe/spire/proto/spire/common/plugin"
+	"github.com/spiffe/spire/test/util"
+	"github.com/stretchr/testify/require"
+	// 	"github.com/spiffe/spire/pkg/server/plugin/upstreamauthority"
+)
+
+func TestGCPCAS(t *testing.T) {
+	ctx := context.Background()
+	fmt.Println("Hi")
+
+	p := New()
+	p.SetLogger(hclog.Default())
```

Comment:
For plugin testing we actually load the plugin using the catalog infrastructure, which wires up the logging and in general provides an easier interface to test against (since you can call the plugin like a normal gRPC client, which is particularly important for plugins with streaming interfaces). See `spiretest.LoadPlugin`.